docs/kits/Business Partner Kit/Documentation BPDM/page_best-practices.md

```diff
@@ -0,0 +1,19 @@
+---
+id: Best Practices
+title: Best Practices
+description: ''
+sidebar_position: 4
+---
+1. Before using the BPDM services, ensure that you have the necessary authentication credentials and permissions to access the services. This may include obtaining an API key or token, and configuring your system to use it.
```

Comment:
Maybe reference OAuth2 authentication flow at this place. The BPDM service currently does not accept API Keys and relies on an oAuth2 resource server to authenticate bearer tokens